packages/node_modules/nav-frontend-skjema/src/radio-panel-gruppe.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { SkjemaGruppe, Fieldset } from './';
+import 'nav-frontend-skjema-style';
+
+export interface RadioProps {
+    label: string;
+    value: string;
+    disabled?: boolean;
+}
+
+export interface RadioPanelGruppeProps {
+    radios: RadioProps[];
+    name: string;
+    legend: string;
+    onChange: (v: string) => void;
+    feil?: object;
+    selectedValue?: string;
+}
+
+export interface RadioPanelGruppeState extends RadioProps {
+    selectedValue?: string;
+}
+
+export interface RadioPanelProps extends RadioProps {
+    checked: boolean;
+    inputId?: string;
+    name: string;
+    onChange: () => void;
+}
+
+interface RadioPanelState {
+    hasFocus: boolean;
+}
+
+class RadioPanel extends React.Component<RadioPanelProps, RadioPanelState> {
+    componentWillMount() {
+        this.setState({ hasFocus: false });
+    }
+
+    toggleOutline() {
+        this.setState({ hasFocus: !this.state.hasFocus });
+    }
+
+    render() {
+        const { checked, disabled, label, name, inputId, onChange } = this.props;
+        const { hasFocus } = this.state;
+
+        const cls = classNames('radioPanel', {
+            'radioPanel--checked': checked === true && !disabled,
+            'radioPanel--focused': hasFocus === true && !disabled,
+            'radioPanel--disabled': disabled === true
+        });
+
+        return (
+            <div
+                className={cls}
+                onClick={() => {
+                    if (!disabled) {
+                        onChange();
+                    }
+                }}
+            >
+                <input
+                    className="radioPanel__radio"
+                    type="radio"
+                    name={name}
+                    checked={checked}
+                    disabled={disabled}
+                    onFocus={() => this.toggleOutline()}
+                    onBlur={() => this.toggleOutline()}
+                />
+                <label className="radioPanel__label" htmlFor={inputId}>{label}</label>
+            </div>
+        );
+    }
+}
+
+class RadioPanelGruppe extends React.Component<RadioPanelGruppeProps, RadioPanelGruppeState> {
+    componentWillMount() {
```

Comment:
`componentWillMount` er på tur ut av react. 
https://medium.com/@baphemot/whats-new-in-react-16-3-d2c9b7b6193b